dashboard/modules/job/cli.py

```diff
@@ -17,12 +17,7 @@ def _get_sdk_client(
     address: Optional[str], create_cluster_if_needed: bool = False
 ) -> JobSubmissionClient:
 
-    if address is None:
-        if "RAY_ADDRESS" not in os.environ:
-            raise ValueError(
-                "Address must be specified using either the --address flag "
-                "or RAY_ADDRESS environment variable."
-            )
+    if address is None and "RAY_ADDRESS" in os.environ:
```

Comment:
Can we add a `logger.info()` statement about picking up the `RAY_ADDRESS` from the environment variable?

Reply:
Ah, just saw the `cli_logger` statement right after this conditional. I'm OK with not including another `logger.info()` statement about the `RAY_ADDRESS` here. I'll leave it up to you.